src/index.js

```diff
@@ -165,6 +172,18 @@ export default function undoable(reducer, rawConfig = {}) {
     default:
       res = reducer(state && state.present, action);
 
+      if (Array.isArray(config.initActions)) {
```

Comment:
What if somebody specifies just one action? (not an array) - maybe at least print a debug output of "initActions ignored because it was an invalid format" or something like that